doc/gromacs.rst

```diff
@@ -108,3 +108,46 @@ The corresponding classes are:
 
     GromacsTopologyFile
     GromacsGroFile
+
+Example usage
+-------------
+
+Many exciting possibilities are available with `GromacsTopologyFile` and `GromacsGroFile`.
+For example, ParmEd is the first software package to enable conversion of Gromacs simulation
+input files to AMBER format in just a few lines of code::
+
+    >>> from chemistry import gromacs, amber
+    >>> gmx_top = GromacsTopologyFile('topol.top')
+    >>> gmx_gro = GromacsGroFile('conf.gro')
+    >>> gmx_top.box = gmx_gro.box # Needed because .prmtop contains box info
+    >>> gmx_top.positions = gmx_gro.positions
+    >>> amb_prm = AmberParm.load_from_structure(gmx_top)
+    >>> amb_prm.write("prmtop")
+    >>> amb_inpcrd = amber.AmberAsciiRestart("inpcrd", mode="w")
+    >>> amb_inpcrd.coordinates = np.array(gmx_top.positions.value_in_unit(u.angstrom)).reshape(-1,3)
```

Comment:
I felt this was less polished than the rest, but I didn't know how else to do it.